ci_framework/roles/devscripts/tasks/03_install.yml

```diff
@@ -0,0 +1,71 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Clone the dev-scripts repository.
+  tags:
+    - bootstrap
+  ansible.builtin.git:
+    repo: "{{ cifmw_devscripts_repo }}"
+    dest: "{{ cifmw_devscripts_repo_dir }}"
+    depth: 1
+    single_branch: true
+    force: true
+    version: HEAD  # noqa: latest[git]
+  register: clone_out
+  retries: 3
+  delay: 15
+  until: "clone_out is not failed"
+
+# Note: Override When external network is being reused.
+- name: Verify dev-scripts vm_setup_vars to reflects external network address
+  tags:
+    - bootstrap
+  when:
+    - "'manage_br_bridge' in cifmw_devscripts_config"
+    - cifmw_devscripts_config['manage_br_bridge'] == 'n'
+  ansible.builtin.replace:
```

Comment:
we may slurp the content, |from_yaml |combine, and re-write the output. Sounds safer.